oms-monitor/docker/docker-compose.yml

```diff
@@ -6,12 +6,19 @@ services:
     image: grafana/grafana:6.4.1
     volumes:
       - grafana-storage:/var/lib/grafana
-      - ./${PATH_OMS_MONITOR}/gf-dashboards:/var/lib/grafana/dashboards
       - ./${PATH_OMS_MONITOR}/config/gf-provisioning:/etc/grafana/provisioning
+      - ./${PATH_OMS_MONITOR}/gf-dashboards/metrics:/var/lib/grafana/dashboards/metrics
+      - ./${PATH_OMS_MONITOR}/gf-dashboards/statistics:/var/lib/grafana/dashboards/statistics
     environment:
       GF_SECURITY_ADMIN_PASSWORD: "${PW_GRAFANA}"
       GF_SERVER_ROOT_URL: "https://${SUBDOMAIN_GRAFANA}${BASE_URL}/"
       GF_INSTALL_PLUGINS: "raintank-worldping-app,grafana-simple-json-datasource,grafana-clock-panel,grafana-piechart-panel"
+      GF_SMTP_ENABLED: "true"
+      GF_SMTP_HOST: "smtp.sendgrid.net:587"
```

Comment:
can we use gsuite for mailing here? I'd go for it instead if possible

Reply:
not sure I agree, why do you propose that?

Reply:
can't recall lol. this can also work